### DISCUSSION TITLE
Left hand (reverse) thread

### DISCUSSION THREAD

**Discussion by uDude** [adrianschlatter/threadlib#53] [General]
Is Appropriate way to get Left threads is mirror, e.g., 

mirror( v = [ 0, 0, -1 ] )   bolt("M8", turns = 20);

???
What about taper?

**Comment by adrianschlatter** [comment]
Hi @uDude 

Yes, by mirroring you get a left-handed thread. However, I'm not sure whether these threads would conform to some standard. As far as I know, the threads currently in threadlib are defined to be _righthanded_ and _untapered_ by their respective norms.
